### ISSUE TITLE
Bug - Currently active tab has no visible :focus or :hover style

### ISSUE THREAD

**Issue by streetdaddy** [AusDTO/gov-au-ui-kit#316]
## What I did

keyboard tabbing through elements of the [Zoo page](http://gov-au-ui-kit-staging.apps.staging.digital.gov.au/examples/zoo.html) when I got to the horizontal tab navigation example.

![gif showing missing :focus style](http://imgur.com/1OxFNK0.gif)
### What browser and device I was using

Testing IE11/Windows 8.1, but this affects all browsers
## What I expected to see

A visible indication that the Department tab was currently focussed
## What I saw

No visible indication

**Comment by klepas** [comment]
Tracked under `DESIGN-405`.
